ow_faults/msg/SystemFaults.msg

```diff
@@ -9,6 +9,7 @@
 # PtGoalError=64, 
 # PtExecutionError=128, 
 # LanderExecutionError = 256
+# PowerExecutionError = 512
 Header header
 uint64 value
```

Comment:
Can you add a simple description to the constants specified in the comments. I do understand them personally but I don't think people new to project might as well.